tests/virt_autotest/reboot_and_wait_up_upgrade.pm

```diff
@@ -48,4 +48,18 @@ sub run {
     $self->reboot_and_wait_up($timeout);
 }
 
+sub enable_y2log_analysis {
+    # Create and Upload y2log for analysis
+    assert_script_run "save_y2logs /tmp/y2logs.tar.bz2", 180;
+    upload_logs("/tmp/y2logs.tar.bz2");
```

Comment:
/root/autoupg.xml is also another important file for offline upgrade, please also find it and upload it.

Reply:
sure, working on it now.
will update the latest validation run about this.

Reply:
quick update, 
had confirmed that there was not autoupg.xml during offline host upgrade:
```
SUSE Linux Enterprise 15 SP2 Installation

Run yast.ssh to start the installation.

2:vh014:~ # pwd
/root
2:vh014:~ # ls -al 
total 8
drwxr-xr-x  3 root root  100 Aug 28 07:58 .
drwxr-xr-x 24 root root  820 Aug 28 07:51 ..
drwx------  3 root root   80 Aug 28 07:47 .gnupg
-rw-r--r--  1 root root 1274 Jun  8 08:25 .profile
-rw-------  1 root root  102 Aug 28 07:58 .Xauthority
2:vh014:~ # ls -al /tmp/
total 12
drwxrwxrwt  7 root root 200 Aug 28 07:51 .
drwxr-xr-x 24 root root 820 Aug 28 07:51 ..
-rw-r--r--  1 root root  29 Aug 28 07:47 host_ips
-rw-r--r--  1 root root  20 Aug 28 07:47 host_name
drwx------  2 root root  40 Aug 28 07:51 libstorage-dvHKGX
-rw-r--r--  1 root root   5 Aug 28 07:47 linuxrc_installer_name
drwxr-xr-x  2 root root  60 Aug 28 07:51 profile
drwxr-xr-x  2 root root  40 Aug 28 07:48 .qt
drwx------ 11 root root 320 Aug 28 07:52 YaST2-21953-LkyPUH
drwx------  4 root root 140 Aug 28 07:52 YaST2-21953-ukMgLk
2:vh014:~ # ls -al 
total 8
drwxr-xr-x  3 root root  100 Aug 28 07:58 .
drwxr-xr-x 24 root root  820 Aug 28 07:51 ..
drwx------  3 root root   80 Aug 28 07:47 .gnupg
-rw-r--r--  1 root root 1274 Jun  8 08:25 .profile
-rw-------  1 root root  102 Aug 28 07:58 .Xauthority
2:vh014:~ # ls -al /
total 432
drwxr-xr-x  24 root root    820 Aug 28 07:51 .
drwxr-xr-x  24 root root    820 Aug 28 07:51 ..
drwxr-xr-x   2 root root   2800 Aug 28 07:47 bin
-rw-r--r--   1 root root  24663 Aug 28 07:47 CHECKSUMS
-rw-r--r--   1 root root    481 Aug 28 07:47 CHECKSUMS.asc
lrwxrwxrwx   1 root root     31 Aug 28 07:51 control.xml -> /usr/lib/skelcd/CD1/control.xml
drwxr-xr-x  12 root root   4960 Aug 28 08:13 dev
drwxr-xr-x   2 root root    260 Aug 28 07:51 download
drwxr-xr-x  42 root root   3200 Aug 28 07:52 etc
-rwxr-xr-x   6 root root 355216 Jun  6 03:46 init
-rw-r--r--   1 root root   2713 Aug 28 07:46 installkey.gpg
-rw-r--r--   1 root root   1757 Aug 28 07:46 installkey.gpg~
lrwxrwxrwx   1 root root     31 Aug 28 07:47 .instsys.config -> /mounts/mp_0001/.instsys.config
drwxr-xr-x   2 root root    140 Aug 28 07:46 lbin
drwxr-xr-x   4 root root    160 Aug 28 07:47 lib
drwxr-xr-x   4 root root   2060 Aug 28 07:47 lib64
-rw-r--r--   1 root root    526 Jun  8 08:25 linuxrc.config
drwxr-xr-x   3 root root     60 Aug 28 07:48 media
drwxr-xr-x   1 root root    188 Aug 28 07:52 mnt
lrwxrwxrwx   1 root root     36 Aug 28 07:46 modules -> lib/modules/5.3.18-22-default/initrd
drwxr-xr-x  19 root root    380 Aug 28 07:51 mounts
drwxr-xr-x   2 root root     40 Aug 28 07:46 oldroot
lrwxrwxrwx   1 root root     30 Aug 28 07:47 .packages.bind -> /mounts/mp_0003/.packages.bind
-rw-r--r--   1 root root   9750 Jun  8 08:25 .packages.initrd
-rw-r--r--   1 root root     36 Jun  8 08:31 .packages.modules
lrwxrwxrwx   1 root root     30 Aug 28 07:47 .packages.root -> /mounts/mp_0001/.packages.root
drwxr-xr-x   4 root root    120 Aug 28 07:46 parts
dr-xr-xr-x 246 root root      0 May 25  2018 proc
-rw-r--r--   1 root root   1274 Jun  8 08:25 .profile
drwxr-xr-x   2 root root    120 Aug 28 07:46 pubkeys
-rw-r--r--   1 root root  13046 Aug 28 07:47 repomd.xml
-rw-r--r--   1 root root    481 Aug 28 07:47 repomd.xml.asc
drwxr-xr-x   3 root root    100 Aug 28 07:58 root
drwxr-xr-x  32 root root    780 Aug 28 08:12 run
drwxr-xr-x   2 root root   4860 Aug 28 07:47 sbin
drwxr-xr-x   3 root root    200 Jun  8 08:25 scripts
dr-xr-xr-x  13 root root      0 May 25  2018 sys
lrwxrwxrwx   1 root root     26 Aug 28 07:47 .timestamp -> /mounts/mp_0001/.timestamp
drwxrwxrwt   8 root root    220 Aug 28 08:13 tmp
drwxr-xr-x   8 root root    180 Aug 28 07:47 usr
drwxr-xr-x   8 root root    200 Jun  8 08:24 var
```